### DISCUSSION TITLE
weird encoding for cyrillic

### DISCUSSION THREAD

**Discussion by y-sands** [watsonbox/exportify#162] [General]
Hiya, fantastic project, thank you so much. I was wondering how to solve a problem - I listen to some songs with Russian song names and when downloaded the playlist csvs, the cyrillic shows up as nonsense letters - e.g. "Ð—Ð°ÐµÐ»Ð° Ð¿ÐµÑÐ½Ñ". I had something similar happen in some html code at some point but I'm not sure how to fix it for this.

**Comment by sharygin** [comment]
Same issue and I considered opening an issue for this but instead will just upvote here. 
I think it is a unicode or other codepage issue because I see it for all non-standard latin characters in spanish, portuguese, german
In the meantime I had success using this page to recover the correct characters for cyrillic and extended latin here: https://2cyr.com/decode/?lang=en

**Comment by watsonbox** [comment]
Could you give me an example playlist that doesn't work? Does this occur when exporting a single playlist?

[Similar reports](https://github.com/watsonbox/exportify/issues/145) have been made in the past that relate more to how the exported files are interpreted, rather than how they are written (UTF8).

**Reply by sharygin** [reply]
Great point, you are right! the exported csv contains the correct utf8 characters, it's an Excel default problem. exportify works great. https://open.spotify.com/playlist/6I8pkxkmJERELi6qfAUY16, #30, #58, #68, #105.
